googletest/src/gtest-printers.cc

```diff
@@ -48,7 +48,7 @@
 #include <ostream>  // NOLINT
 #include <string>
 #include "gtest/internal/gtest-port.h"
-#include "src/gtest-internal-inl.h"
+#include "gtest-internal-inl.h"
```

Comment:
How is this related to the bazel change? Please revert and just leave the bazel related changes.